### DISCUSSION TITLE
Disable DJ Autorecord?

### DISCUSSION THREAD

**Discussion by Radiotom** [AzuraCast/AzuraCast#7273] [Q&A]
Hi Folks,

i installed AC on a vanialla System: 
[AzuraCast](https://www.azuracast.com/) • Rolling Release #[b5c5c22] (2024-07-02 0:43) • Docker • PHP 8.3

We use AzuraCast as a Bridge between oure Radiobroadcast Software mAirList and oure Radioservice (Laut.Fm)
The mAirList 24/7 Signal is Setup as a DJ and Broadcast directly to AzuraCast.
Next Step is a Remote Relay to Laut.fm Server. 

That works very well.
If the mAirList Broadcast disabled, Liquidsoap jumps into the main Signal as AutoDJ. 

My issue today is:
The "DJ" mAirlist, who sends 24/7 are recordet. 

Where i can disable the record funktion?
Or who can i setup the external mAirList 24/7 Signal not as a DJ?

Thank u :-)

**Comment by Radiotom** [comment]
i allreday disabled this Option but it had no Effekt. 
![image](https://github.com/AzuraCast/AzuraCast/assets/118381734/55642d2c-57a2-444a-b767-364f35ea6f50)

The Station still record the Live Signal ?
![image](https://github.com/AzuraCast/AzuraCast/assets/118381734/74efc200-7bf4-4bd5-9e8e-a5b158d7b664)

**Reply by LordHelmchen666** [reply]
It is not recording as a file. It is just logging the DJ has connected at this time. If it would recording to a file, there would be also a link to listen and download the file.  ;)

**Comment by bombelman** [comment]


**Reply by Radiotom** [reply]
How can i use without DJ a ist external 24/7 source without disable the AutoDJ?

The AutoDJ is oure "Fallback" System. Wo dont like the AutoDJ of LautFM 😋

**Comment by Radiotom** [comment]
I will not disable the AutoDJ Funktion because thats oure Fallback System if mAirList Not sends any Data.

I only want to deaktivate the record Funktion of the DJ Source. 🤷 But it still record the live Stream.

**Reply by bombelman** [reply]
I understand better now😉

**Comment by Moonbase59** [comment]
@Radiotom A mAirList user, how neat! I can assure you AzuraCast works fine with mAirList.

Look here for the difference between a _log entry_ and an _actual recording_:

![Nite Radio – Mozilla Firefox_044](https://github.com/AzuraCast/AzuraCast/assets/3706922/55005bec-cc94-4062-adf7-073c5d5257eb)

**Reply by Radiotom** [reply]
Aaaaaah, omg. Thank u. 
I had to Check that at Home. 😋
